contracts/connectors/loantoken/LoanTokenSettingsLowerAdmin.sol

```diff
@@ -20,24 +20,16 @@ contract LoanTokenSettingsLowerAdmin is AdvancedToken {
 	/// ------------- MUST BE THE SAME AS IN LoanToken CONTRACT -------------------
 	address public sovrynContractAddress;
```

Comment:
We don't need to maintain storage compatibility since LoanTokens (proxy contracts) were already deployed and we need to add variables only to LoanTokenSettingsLowerAdmin. 
I prefer to leave as is for now.